iojs.org/aliases.json

```diff
@@ -58,30 +66,40 @@
     "rtrott@gmail.com",
     "thechargingvolcano@gmail.com"
   ] },
-  { "from": "admin", "to": [
+  { "from": "admin",
+    "description": "build/infra team, used primarily for authenticating domain-level ownership",
+    "to": [
     "jbergstroem@iojs.org",
     "michael_dawson@ca.ibm.com",
     "reis@janeasystems.com",
     "rod@vagg.org"
   ] },
-  { "from": "accounts", "to": [
+  { "from": "accounts",
+    "description": "build/infra team, used for infra-level accounts with various external services",
+    "to": [
     "jbergstroem@iojs.org",
     "michael_dawson@ca.ibm.com",
     "reis@janeasystems.com",
     "rod@vagg.org"
   ] },
-  { "from": "build", "to": [
+  { "from": "build",
+    "description": "build/infra team, used for infra-level accounts with various external services (prefer accounts@ for new accounts)",
```

Comment:
Do we need to explain what it is, can't we just say it's a deprecated alias (or equivalent) to accounts@ and leave it at that?